test/mocha.opts

```diff
@@ -1,6 +1,6 @@
 --ui tdd
 --require test/helper
---timeout 240s
+--timeout 480s
```

Comment:
Ah, maybe we could find out a way to just set the timeout for this one test.  Maybe there is a suite.timeout() method in mocha rather than enabling it globally.